src/ceph-disk/ceph_disk/main.py

```diff
@@ -777,7 +803,7 @@ def get_partition_base(dev):
     Get the base device for a partition
     """
     dev = os.path.realpath(dev)
-    if not stat.S_ISBLK(os.lstat(dev).st_mode):
+    if ldev_is_diskdevice(dev):
```

Comment:
should be 
```py
if not ldev_is_diskdevice(dev):
```

```
2017-07-14T04:09:31.086 INFO:tasks.workunit.client.0.vpm021.stderr:DEBUG:CephDisk:Traceback (most recent call last):
2017-07-14T04:09:31.088 INFO:tasks.workunit.client.0.vpm021.stderr:DEBUG:CephDisk:File "/usr/sbin/ceph-disk", line 9, in <module>
2017-07-14T04:09:31.091 INFO:tasks.workunit.client.0.vpm021.stderr:DEBUG:CephDisk:load_entry_point('ceph-disk==1.0.0', 'console_scripts', 'ceph-disk')()
2017-07-14T04:09:31.113 INFO:tasks.workunit.client.0.vpm021.stderr:DEBUG:CephDisk:File "/usr/lib/python2.7/dist-packages/ceph_disk/main.py", line 5725, in run
2017-07-14T04:09:31.116 INFO:tasks.workunit.client.0.vpm021.stderr:DEBUG:CephDisk:main(sys.argv[1:])
2017-07-14T04:09:31.125 INFO:tasks.workunit.client.0.vpm021.stderr:DEBUG:CephDisk:File "/usr/lib/python2.7/dist-packages/ceph_disk/main.py", line 5676, in main
2017-07-14T04:09:31.127 INFO:tasks.workunit.client.0.vpm021.stderr:DEBUG:CephDisk:args.func(args)
2017-07-14T04:09:31.130 INFO:tasks.workunit.client.0.vpm021.stderr:DEBUG:CephDisk:File "/usr/lib/python2.7/dist-packages/ceph_disk/main.py", line 4034, in main_destroy
2017-07-14T04:09:31.198 INFO:tasks.workunit.client.0.vpm021.stderr:DEBUG:CephDisk:main_destroy_locked(args)
2017-07-14T04:09:31.236 INFO:tasks.workunit.client.0.vpm021.stderr:DEBUG:CephDisk:File "/usr/lib/python2.7/dist-packages/ceph_disk/main.py", line 4061, in main_destroy_locked
2017-07-14T04:09:31.239 INFO:tasks.workunit.client.0.vpm021.stderr:DEBUG:CephDisk:base_dev = get_partition_base(dev_path)
2017-07-14T04:09:31.242 INFO:tasks.workunit.client.0.vpm021.stderr:DEBUG:CephDisk:File "/usr/lib/python2.7/dist-packages/ceph_disk/main.py", line 828, in get_partition_base
2017-07-14T04:09:31.244 INFO:tasks.workunit.client.0.vpm021.stderr:DEBUG:CephDisk:raise Error('not a block device', dev)
2017-07-14T04:09:31.250 INFO:tasks.workunit.client.0.vpm021.stderr:DEBUG:CephDisk:ceph_disk.main.Error: Error: not a block device: /dev/vdb1
```

/a/kchai-2017-07-14_03:23:35-ceph-disk-wip-15432-kefu-distro-basic-vps/1398707/teuthology.log